src/components/Buttons/SubmitButton.jsx

```diff
@@ -14,7 +15,29 @@ const loadingColors = {
     danger: "!bg-danger-light",
 };
 
-export const SubmitButton = ({ color = "primary", isLoading, className, children, ...rest }) => {
+export const SubmitButton = ({ color = "primary", isLoading, isSuccess, className, children, ...rest }) => {
+    const [showSuccess, setShowSuccess] = useState(false);
+
+    useEffect(() => {
+        let timer = null;
+        if (isSuccess && !isLoading) {
+            setShowSuccess(true);
+            timer = setTimeout(() => setShowSuccess(false), 3000);
```

Comment:
We can make this cleaner:

```suggestion
            const timer = setTimeout(() => setShowSuccess(false), 3000);
            return () => clearTimeout(timer);
```

Now you can remove the `let timer = null` at the top